modules/phase_field/src/kernels/CHParsed.C

```diff
@@ -54,13 +56,13 @@ CHParsed::computeGradDFDCons(PFFunctionType type)
   {
     case Residual:
       for (unsigned int i = 0; i <= _nvar; ++i)
-        res += (*_grad_vars[i])[_qp] * (*_second_derivatives[i])[_qp];
+        res += (*_grad_vars[i])[_qp]*(*_second_derivatives[i])[_qp];
       return res;
 
     case Jacobian:
-      res = _grad_phi[_j][_qp] * (*_second_derivatives[0])[_qp];
+      res = _grad_phi[_j][_qp]*(*_second_derivatives[0])[_qp];
       for (unsigned int i = 0; i <= _nvar; ++i)
-        res += _phi[_j][_qp] * (*_grad_vars[i])[_qp] * (*_third_derivatives[i])[_qp];
+        res += _phi[_j][_qp]*(*_grad_vars[i])[_qp]*(*_third_derivatives[i])[_qp];
```

Comment:
I like to space the math operators in terms of order of operations. So, I have no space around the * or / and I leave a space on either side of the + and -. It makes it much easier for me to quickly parse when I look at a line of math.